CONTRIBUTING.md

```diff
@@ -0,0 +1,21 @@
+# Contribution Guidelines
+
+_Welcome hackers it is really **awesome** have you here! Before start contributing with this project make sure you read our [Code Of Conduct](https://github.com/0x48piraj/Awesome-Hacktoberfest/blob/master/CODE_OF_CONDUCT.md), it is really important to make this inclusive and open to everyone, otherwise, it would not be awesone_
+
+## Have you find some awesome Hacktoberfest participant repository?
```

Comment:
Add something like : "Add your achievements! What you did at this Hacktoberfest? Write it down, don't just do and forget."